pages.fr/common/git-clone.md

```diff
@@ -0,0 +1,24 @@
+# git clone
+
+> Clone un repository existant.
+> Plus d'informations : <https://git-scm.com/docs/git-clone>.
+
+- Clone un repository existant :
+
+`git clone {{remote_repository_location}}`
+
+- Clone un repository existant et ses sous-modules : 
+
+`git clone --recursive {{remote_repository_location}}`
+
+- Pour cloner depuis une machine local :
```

Comment:
This was badly worded in English. Please take a look at [the english page](https://github.com/tldr-pages/tldr/blob/master/pages/common/git-clone.md) and translate this accordingly:

    - Clone a local repository: